user-service/.env.example

```
@@ -21,5 +21,14 @@ PORTONE_API_SECRET=
 ENCRYPTION_SECRET=
 
-#NICKNAME
+# Nickname
 NICKNAME_PREFIX=
+
+# Internal API Secret
+INTERNAL_API_SECRET=
+
+# Service URLs (optional - defaults to localhost in local)
+DRAW_SERVICE_URL=
+AUCTION_SERVICE_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AUCTION_SERVICE_URL key should go before the DRAW_SERVICE_URL key

(UnorderedKey)